### PR TITLE
Updated Radio List Buttons

### PR DESCRIPTION
**Pull request by ninjaofawesome**
# OVERVIEW
this PR covers the stacked radio buttons found throughout the site, such as on sidebars.  The colors have been updated to vimeo blue instead of VHX-teal

screen shot 2017-09-27 at 3 00 13 pm

# SOLUTION
- Removed teal option from CSS styling, replaced with vimeo-blue
- Updated ternary operators, `radio.ui.js`

### CODE REVIEW

File: dist/component-temp.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
+/*
+Error: Undefined variable: "$vimed-blue-darkened".
```
Comment (davegonzalez) [inline dist/component-temp.css]: Seems like a typo somewhere is causing a build error?

**Comment by davegonzalez** [comment]
What is `component-temp.js` for?

File: dist/component-temp.css (hunk 1 above)
Comment (ninjaofawesome) [reply dist/component-temp.css]: I corrected it and rebuilt, this could have been residual

Comment (ninjaofawesome) [reply dist/component-temp.css]: looks like its just a log, not critical, will remove and add.

**Pull request merged**